core/src/commonMain/kotlin/com/xebia/functional/xef/llm/openai/OpenAIEmbeddings.kt

```diff
@@ -22,7 +22,7 @@ class OpenAIEmbeddings(private val config: OpenAIConfig, private val oaiClient:
     }
     val lists: List<List<Embedding>> =
       if (texts.isEmpty()) emptyList()
-      else texts.chunked(chunkSize ?: config.chunkSize).parMap { createEmbeddings(it) }
+      else texts.chunked(chunkSize ?: 400).parMap { createEmbeddings(it) }
```

Comment:
Why this introduction of a magic number, shouldn't the default exist inside of `OpenAIConfig` instead?

Reply:
It does not look like it's available by the `OpenAIConfig` provided by the library.

Reply:
@diesalbla This is the code location we discussed regarding the magic number in today's meeting.